block/manager.go

```diff
@@ -39,7 +39,7 @@ type Manager struct {
 	// Store and execution
 	store     store.Store
 	lastState types.State
-	executor  *state.BlockExecutor
+	executor  *BlockExecutor
 
```

Comment:
Seeing as you are already refactoring, I think it would be nice now to remove the Block prefix from types in this package

For example, BlockExecutor should now just be Executor

Reply:
what do you think @mtsitrin ?